### ISSUE TITLE
OCS module uses cluster-monitoring-config ConfigMap in openshift-monitoring to create PVs

### ISSUE THREAD

**Issue by netzzer** [openshift/openshift-cns-testdrive#484]
OCS module requires modification to ConfigMap cluster-monitoring-config with addition of content found in this yaml file - https://github.com/red-hat-storage/ocs-training/blob/master/ocp4ocs4/cluster-monitoring-config.yaml.

The OCS ConfigMap content needs to be added to ConfigMap content found in this yaml file - https://github.com/openshift/openshift-cns-testdrive/blob/ocp4-dev/support/cluster-monitoring-configmap.yaml.

**Comment by netzzer** [comment]
Need to add Warning similar to below and either edit or patch existing ConfigMap created using https://github.com/openshift/openshift-cns-testdrive/blob/ocp4-dev/support/cluster-monitoring-configmap.yaml.

```
[WARNING]
====
This section requires that you have completed the infra-nodes lab. 
====
```

**Comment by mulbc** [comment]
There's no real dependency here... 
There are two scenarios:

if the student does infra first and the OCS section overwrites the ConfigMap (CM):

* Prometheus Pods are recreated with OCS-backed disks
* Prometheus Pods are able to schedule on all *schedulable* Nodes

if the student does OCS first and the infra section overwrites the ConfigMap (CM):

* Prometheus Pods are recreated with epemeral disks
* Prometheus Pods are able to schedule only on the infra Nodes

We just have to make sure that `oc apply` is used instead of `oc create` so that over-writing will be possible

**Comment by mulbc** [comment]
@thoraxe - please review commit 7400d17 (see above this comment) and merge if you agree to close this issue

**Comment by netzzer** [comment]
Fixed with PR #492 

Do NOT merge commit 7400d17
